examples/nginxproxy-example/nginxproxy-error-loglevel.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: gateway.nginx.org/v1alpha1
```

Comment:
I think we can remove this since we have working examples in the documentation you added